docs/content/misc/math-expr.md

```diff
@@ -195,3 +195,11 @@ See javadoc of java.lang.Math for detailed explanation for each function.
 | cartesian_fold(lambda,arr1,arr2,...) | folds a multi argument lambda across the cartesian product of all input arrays. The first arguments of the lambda is the array element and the last is the accumulator, returning a single accumulated value. |
 | any(lambda,arr) | returns 1 if any element in the array matches the lambda expression, else 0 |
 | all(lambda,arr) | returns 1 if all elements in the array matches the lambda expression, else 0 |
+
+
+## IP Address Functions
+| function | description |
+| --- | --- |
+| ipv4address_match(address, subnet [,inclusive]) | Returns 1 if the string or long IP `address` belongs to the `subnet` CIDR notation literal string, else 0. The optional literal boolean `inclusive` parameter indicates whether the network and broadcast IP addresses should be included in the subnet and defaults to `false`. This function is more efficient when operating on IP addresses as longs instead of strings.|
```

Comment:
The network and broadcast addresses aren't usable IP addresses in the subnet, which is why I excluded them by default. However, since they _do_ match the subnet, I'll change the default to include them.